osm2lanes/src/transform/tags_to_lanes/modes/bicycle.rs

```diff
@@ -29,17 +29,57 @@ impl From<VariantError> for TagsToLanesMsg {
 
 struct Opposite;
 
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub(in crate::transform::tags_to_lanes) enum Variant {
+    SharedMotor,
+    // SharedBus,
+    // OptionalLane,
+    Lane,
+    Track,
+}
+
+impl Display for Variant {
+    #[allow(clippy::todo, clippy::panic_in_result_fn)]
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                Self::SharedMotor => todo!(),
+                Self::Lane => "lane",
+                Self::Track => "track",
+            }
+        )
+    }
+}
+
+#[derive(Debug, Clone, PartialEq)]
+pub(in crate::transform::tags_to_lanes) enum OptionNo<T> {
```

Comment:
A weird one 😅

Reply:
I didn't think it needed a comment, but I did need to differentiate missing from no